index.js

```diff
@@ -11,6 +11,7 @@ function CompressionPlugin(options) {
 	options = options || {};
 	this.asset = options.asset || "[path].gz[query]";
 	this.algorithm = options.algorithm || "gzip";
+	this.changeNewFileName = options.changeNewFileName || false;
```

Comment:
Ah, yeah... We would have to update this project to **webpack-defaults** to get testing infrastructure in place.